### ISSUE TITLE
Validation UI

### ISSUE THREAD

**Issue by cchristelis** [kartoza/healthyrivers#142]
# Problem

Currently our valuidation workflow is breaking the rest of our UI.

# Proposed Solution

Design this to fit in with the rest of the site's look and feel. I think it might even be worth the while to do validation directly on the map? Either way please go through the workflow and update the mockups.

**Comment by gubuntu** [comment]
Doing it on the map was one of my initial suggestions for LEDET. I like it because it means staying on the map page (validation records can be in a side panel) and having all the existing map layers and functionality available rather than having to work them into another page. A validator needs access to multiple layers to assist with the validation process. 

So any improvements here should go into django-bims (except for branding skinning)

**Comment by dimasciput** [comment]
@gubuntu But what if there are >1000 data? I think table is better for validator, because then they can sort, search and do other stuff from the table.

**Comment by gubuntu** [comment]
yes, a table on the main map page, rather than a map in a separate view. We need a table in the map page for other projects too, so it would help to do it that way here.

**Comment by cchristelis** [comment]
@dimasciput, the normal case would not have > 1000 data points unvalidated. The initial ingestion of data we'll validate programmatically anyway, (At least in the case of healthyrivers) since we know that the initial import data has been vetted. So this feature will usually only contain a managable amount of data none the less +1 on having it in a table in the UI.

**Comment by dimasciput** [comment]
@cchristelis 
- And what is user uploads bulk of data in csv? Like hundreds data? Is that possible? 
- And also we only display location site on the map, what if there is like 10 collection data on one location site?

**Comment by gubuntu** [comment]
implemented in map view
